lib/src/login/login_provider_list.dart

```diff
@@ -94,10 +97,10 @@ class _ProviderListState extends State<ProviderList> {
   @override
   Widget build(BuildContext context) {
     if (widget.type == ProviderListType.login) {
-      title = L10n.get(L.loginSignIn);
+      title = ProviderList.loginViewTitle;
       text = L10n.get(L.loginChooseProvider);
     } else if (widget.type == ProviderListType.register) {
-      title = L10n.get(L.register);
+      title = ProviderList.registerViewTitle;
```

Comment:
And btw: For consistency reasons we should use either `L.loginChooseProvider` and `L.registerChooseProvider` or `L.chooseProviderLogin` and `L.chooseProviderRegister`.

Reply:
Make sense